test/.eslintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "globals": {
+    "Java": "writable"
```

Comment:
For test execution, the global `Java` needs to be writable, because in `java.mock.js` I have to (re)set it with a mock object. In to root `.eslintrc` `Java` is restricted to be read only. Either I have to override this in `test/.eslintrc`, as I have done, or I have to disable this ESLint rule in `java.mock.js`:

```js
/* eslint-disable-next-line no-global-assign */
Java = {
  type: (type) => TYPES[type],
  from: function () {},
  isType: function () {}
};
```

I've no preference, which way to go.

Regarding the rule in `.eslintrc` to enforce semicolons: ESLint merges all config files it finds, starting with the current directory, all the way up. So the rules in `.eslintrc` will be valid for the files in the `test` directory, too.

Reply:
> or I have to disable this ESLint rule in java.mock.js:

I would prefer this way, as it is less confusing IMO. 

> ESLint merges all config files it finds, starting with the current directory, all the way up. So the rules in .eslintrc will be valid for the files in the test directory, too.

Okay, thanks for the explanation 👍

Reply:
I moved the disabling of the rule to `java.mock.js`.